src/app/@ansyn/ansyn/modules/menu-items/cases/components/edit-case/edit-case.component.ts

```diff
@@ -89,6 +89,9 @@ export class EditCaseComponent implements OnInit, OnDestroy {
 	}
 
 	getEmptyCase(selectedCase: ICase): ICase {
+		if (!selectedCase) {
```

Comment:
what is this fixing?

Reply:
@Danaorlov1005 There was a crash when I send the action to close the dialog. The addition prevents the crash